docs/resources/cbh_instance.md

```diff
@@ -62,6 +62,11 @@ The following arguments are supported:
 
   Changing this parameter will create a new resource.
 
+* `product_info` - (Required, List, ForceNew) Specifies the product info of the instance.
```

Comment:
the `product_info` has marked be required, but it does not used in example above